### PR TITLE
Feature/waiting for messages from component via LM

### PR DESCRIPTION
**Pull request by jungeunyoon**
How we're going to facilitate components wasn't really mentioned in the design docs so added a components node module to start us off. We can discuss more as we move onto actually sending messages to LS.

### CODE REVIEW

**Comment by jungeunyoon** [comment]
[Implement waiting for messages [2]](https://trello.com/c/o7QBshZD/4531-implement-waiting-for-messages-2)

**Comment by jungeunyoon** [comment]
[Implement waiting for messages [2]](https://trello.com/c/o7QBshZD/4531-implement-waiting-for-messages-2)

File: src/messaging.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  } catch (error) {
+    logger.file(`message recieved error - ${config.moduleName} - ${error}`);
+  }
+}
+
+function handleFileUpdate(message) {
+  if (!message.filePath || !message.filePath.startsWith(companyConfigBucket)) {
+    return;
+  }
+  if (message.filePath.endsWith("/twitter.txt")) {
```
Comment (rodrigopavezi) [inline src/messaging.js]: @jungeunyoon a small thing. I changed it to twitter.json as the content is actually json. cheers

Comment (jungeunyoon) [reply src/messaging.js]: @rodrigopavezi good catch! fixed, thanks.

**Comment by jungeunyoon** [comment]
@rodrigopavezi merge conflicts addressed. good for re-review, thanks!

**Comment by rodrigopavezi** [comment]
@jungeunyoon looks good 👍

**Pull request merged**